goal-planning/goal-planning-frontend
# Chore / Add Overflow Ellipse to User Name Text on Navbar

## Changes
1. Updated navbar CSS to provide text overflow ellipse.

## Purpose
Resolve edge case scenarios where users may have too long of a name that will distort the UI renderings.

## Approach
Find the container hold the desire text to perform overflow ellipse and apply the necessary CSS. Which in this case, is an overflow hidden style followed by a text overflow ellipse. Then the width is there to identify the boundary limits of the container holding the text.

<img width="703" alt="Screenshot 2023-06-09 at 3 22 48 PM" src="https://github.com/goal-planning/goal-planning-frontend/assets/31714174/c37271b2-0cd5-4394-abaf-2260cd428eb9">
<img width="703" alt="Screenshot 2023-06-09 at 3 22 56 PM" src="https://github.com/goal-planning/goal-planning-frontend/assets/31714174/264a3995-a7fa-4371-bc27-aae33d75df62">


## Testing Steps
1. Pull down code.
2. Run the application and register a user the has a really long name.
3. Log in as that user with the very long name.
4. Confirm that the user's name on the navbar was overflowed ellipse. Also inspect the source and view in mobile mode.

Closes #57
